fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/RootRdfContext.java

```diff
@@ -129,7 +132,25 @@ private void concatRepositoryTriples() {
                     createTypedLiteral(counters.get(PREFIX + FIXITY_REPAIRED_COUNTER).getCount()).asNode()));
         }
 
+        //add project build description triples here
+        final BuildPropertiesLoader buildLoader = new BuildPropertiesLoader();
+
+        final com.hp.hpl.jena.graph.Node releaseSubject = createURI(REPOSITORY_NAMESPACE + "release");
+            b.add(create(subject(), DOAP_RELEASE.asNode(), releaseSubject));
+            b.add(create(releaseSubject, type.asNode(), createURI(DOAP_NAMESPACE + "Version")));
```

Comment:
Circling back on this ticket, it seems that the solution that is being sought is a simple predicate on the /root of the repository that holds the property of the Fedora release version number. Given the apparent lack of suitable existing terms ([doap](http://usefulinc.com/ns/doap) properties seems to have a range of doap:Project or doap:Version which would require new Fedora resources) would it instead make sense to create a new fedora:hasReleaseNumber property to http://fedora.info/definitions/v4/repository# ?